services/movie.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: List[int] = None, actors_ids: List[int] = None):
```

Comment:
```suggestion
def get_movies(genres_ids: list[int] = None, actors_ids: list[int] = None):
```

use python 3.10+)

Reply:
finally found a reason to update from my 3.7

Reply:
))